io-ballerina/src/io/character/channel_string_io.bal

```diff
@@ -183,14 +183,14 @@ public function channelWriteLinesFromStream(WritableChannel writableChannel, str
     }
 }
 
-# Write a JSON to a channel.
+# Write a JSON to a channel
 # ```ballerina
 # json content = {"name": "Anne", "age": 30};
 # io:Error? content = io:channelWriteJson(writableChannel);
 # ```
-# + writableChannel - A writable channel
+# + writableChannel - A writable channel. The possible inputs are `io:WritableByteChannel` or `io:WritableCharacterChannel`.
 # + content - JSON content to write
-# + return - `io:Error` or else `()`
+# + return - Either an `io:Error` or the null `()` value when the writing was successful
```

Comment:
Same issue in the description

Reply:
Fixed in here https://github.com/ballerina-platform/module-ballerina-io/pull/37/commits/123095180ab4fe7865b5e69d8623895fe6b978e8